src/main/java/io/github/zekerzhayard/forgewrapper/installer/detector/IFileDetector.java

```diff
@@ -20,6 +19,29 @@ public interface IFileDetector {
      */
     boolean enabled(HashMap<String, IFileDetector> others);
 
+
+    default URL getClassURL() {
```

Comment:
I don't think makes sense to be a separate method in the interface that can be called, overriden, etc. so I think it would make sense to move it into a lambda in getLibraryDir (I don't think you can do private static in an interface in java 8)